<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Fraeser_Aufruf</class>
 <widget class="QDialog" name="Dialog_Fraeser_Aufruf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fraeser-Aufruf</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>allgemein</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>561</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0,0">
      <item row="6" column="0">
       <widget class="QLabel" name="label_kantdi">
        <property name="text">
         <string>Kantendicke</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_startz">
        <property name="text">
         <string>Startpunkt Z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_startx">
        <property name="text">
         <string>Startpunkt X</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_starty">
        <property name="text">
         <string>Startpunkt Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_starty_var">
        <property name="text">
         <string>*[Y]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox_werkzeug"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label__bahnkor_var">
        <property name="text">
         <string>[BKO]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBox__bahnkor"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_werkzeug">
        <property name="text">
         <string>Werkzeug</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label__werkzeug_var">
        <property name="text">
         <string>[WKZ]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_bahnkor">
        <property name="text">
         <string>Bahnkorrektur</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_startz_var">
        <property name="text">
         <string>*[Z]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_eckruglob">
        <property name="text">
         <string>Eckenrunden Global</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label__eckruglob_var">
        <property name="text">
         <string>*[ERG]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_starty"/>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_kantdi_var">
        <property name="text">
         <string>[KD]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lineEdit_eckruglob"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_startz"/>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="lineEdit_kantdi"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_startx"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label__startx_var">
        <property name="text">
         <string>*[X]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_bild_fraeser">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>315</y>
       <width>281</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string>Bild des Fräsers</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Technologie</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>7</y>
       <width>551</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0">
       <widget class="QLabel" name="label_abftyp">
        <property name="text">
         <string>Abfahrtyp</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_ausfbed_var">
        <property name="text">
         <string>*[AFB]</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="lineEdit_ausfbed"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_bezeichn">
        <property name="text">
         <string>Bezeichnung</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_komment">
        <property name="text">
         <string>Kommentar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_vorschub">
        <property name="text">
         <string>Vorschub XY</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_ausfbed">
        <property name="text">
         <string>Ausführbedingung</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="lineEdit_bezeichn"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_drehzahl">
        <property name="text">
         <string>Drehzahl</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_komment_var">
        <property name="text">
         <string>[KOM]</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_bezeichn_var">
        <property name="text">
         <string>[BEZ]</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="lineEdit_komment"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_anftyp">
        <property name="text">
         <string>Anfahrtyp</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_eintvors">
        <property name="text">
         <string>Eintauchvorschub</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_abftyp_var">
        <property name="text">
         <string>[ANT]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_anftyp_var">
        <property name="text">
         <string>[ABT]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_drehzahl_var">
        <property name="text">
         <string>[N]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_vorschub_var">
        <property name="text">
         <string>[F]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_eintvors_var">
        <property name="text">
         <string>[FAN]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_eintvors"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_vorschub"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_drehzahl"/>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="comboBox_anftyp"/>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="comboBox_abftyp"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label__zustellung">
        <property name="text">
         <string>Zustellmass</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_zustellung_var">
        <property name="text">
         <string>[ZUST]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_zustellung"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>620</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_speichern">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_werkzeug</tabstop>
  <tabstop>comboBox__bahnkor</tabstop>
  <tabstop>lineEdit_startx</tabstop>
  <tabstop>lineEdit_starty</tabstop>
  <tabstop>lineEdit_startz</tabstop>
  <tabstop>lineEdit_eckruglob</tabstop>
  <tabstop>lineEdit_kantdi</tabstop>
  <tabstop>lineEdit_zustellung</tabstop>
  <tabstop>lineEdit_eintvors</tabstop>
  <tabstop>lineEdit_vorschub</tabstop>
  <tabstop>lineEdit_drehzahl</tabstop>
  <tabstop>comboBox_anftyp</tabstop>
  <tabstop>comboBox_abftyp</tabstop>
  <tabstop>lineEdit_komment</tabstop>
  <tabstop>lineEdit_bezeichn</tabstop>
  <tabstop>lineEdit_ausfbed</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
  <tabstop>pushButton_speichern</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
